Sources/PackageDescription4/SupportedPlatforms.swift

```diff
@@ -33,6 +33,9 @@ public struct Platform: Encodable {
 
     /// The Linux platform.
     public static let linux: Platform = Platform(name: "linux")
+
+    /// The Windows platform
+    public static let windows: Platform = Platform(name: "windows")
```

Comment:
We'll need to gate this using the package description availability. I suggest proposing this change separately. I don't think we need a proposal for this but I'll check with some other SwiftPM contributors.

Reply:
Proposed separately in https://github.com/apple/swift-package-manager/pull/2382